docs/source/skeleton/index.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../../../README.rst
```

Comment:
We should add some comments above this line telling the user to modify this index file content.

Reply:
Sure! Makes more sense.